### PR TITLE
DMP-1290: Set up Role Permission for External Services

### PR DESCRIPTION
**Pull request by hemantasharma1129**
**Before creating a pull request make sure that:**

- [ x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [ x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DMP-1290

### Change description ###
Data Setup for External Services - XHIBIT, CPP, DAR_PC & MID_TIER

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by karen-hedges** [comment]
Do you need to add DAR_MIDTIER_USER_NAME and DAR_MIDTIER_PASSWORD to docker-compose-local.yml?

**Comment by hemantasharma1129** [comment]
@karen-hedges yes we would need it in docker-compose yaml

File: src/main/resources/db/migration/common/V1_227__insert_security_role_permission_external_services.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+insert into security_role_permission_ae(rol_id,per_id) values (7,16);
+insert into security_role_permission_ae(rol_id,per_id) values (8,16);
+insert into security_role_permission_ae(rol_id,per_id) values (9,17);
+insert into security_role_permission_ae(rol_id,per_id) values (9,18);
+insert into security_role_permission_ae(rol_id,per_id) values (10,18);
+insert into security_role_permission_ae(rol_id,per_id) values (10,19);
+insert into security_role_permission_ae(rol_id,per_id) values (10,20);
+insert into security_role_permission_ae(rol_id,per_id) values (10,21);
+
+ALTER SEQUENCE rol_seq RESTART WITH 10;
```
Comment (richardbull) [inline src/main/resources/db/migration/common/V1_227__insert_security_role_permission_external_services.sql:22]: rol_seq should be 11? seems to be 1 short

File: src/main/resources/db/migration/common/V1_227__insert_security_role_permission_external_services.sql [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ALTER SEQUENCE per_seq RESTART WITH 23;
```
Comment (richardbull) [inline src/main/resources/db/migration/common/V1_227__insert_security_role_permission_external_services.sql:23]: 24?

File: src/main/resources/db/migration/common/V1_227__insert_security_role_permission_external_services.sql (hunk 1 above)
Comment (hemantasharma1129) [reply src/main/resources/db/migration/common/V1_227__insert_security_role_permission_external_services.sql]: Updated.

File: src/main/resources/db/migration/common/V1_227__insert_security_role_permission_external_services.sql (hunk 2 above)
Comment (hemantasharma1129) [reply src/main/resources/db/migration/common/V1_227__insert_security_role_permission_external_services.sql]: Updated.

**Review by karen-hedges — APPROVED** [review]

**Comment by richardbull** [comment]
@hemantasharma1129 The JIRA mentions security_group for the external processing user - but I don't see any inserts into these tables?

**Review by richardbull — APPROVED** [review]

**Pull request merged**